File: JAVA/circle.java
package circle.java;
import java.util.Scanner;

public class circle{
   public static void main(String[] args){
      Scanner scanner = new Scanner(System.in);
      
      double radius;
      double area;
      double circumference;
      double diameter;
      
      System.out.print("Enter a radius: ");
      if(!scanner.hasNextDouble()){
         System.out.println("Radius must be a number!");
      }
      radius = scanner.nextDouble();
      
      System.out.println("");
      System.out.printf("Circumference: %,.2fcm\n", circumference(radius));
      System.out.printf("Area: %,.2fcm²\n", area(radius));
      System.out.printf("Diameter: %,.2fcm\n", diameter(radius));
      
   }
   static double circumference(double radius){
      double circumference = 2 * Math.PI * radius;
      return circumference;
   }
   static double area(double radius){
      double area = Math.PI * Math.pow(radius, 2);
      return area;
   }
   static double diameter(double radius){
      double diameter = 2 * radius;
      return diameter;
   }
}

File: C/circle.c
#include <stdio.h>
#include <math.h>

int main(){
   
   double radius = 0.0;
   double area = 0.0;
   double surfaceArea = 0.0;
   double circumference = 0.0;
   double diameter = 0.0;
   double volume = 0.0;
   const double PI = 3.14159265359;
   
   printf("Enter a radius: ");
   scanf("%lf", &radius);
   
   area = PI * pow(radius, 2);
   surfaceArea = 4 * PI * pow(radius, 2);
   circumference = 2 * PI * radius;
   diameter = 2 * radius;
   volume = (4.0 / 3.0) * PI * pow(radius, 3);
   
   printf("\nAREA: %.2lf\n", area);
   printf("SURFACE AREA: %.2lf\n", surfaceArea);
   printf("CIRCUMFERENCE: %.2lf\n", circumference);
   printf("DIAMETER: %.2lf\n", diameter);
   printf("VOLUME: %.2lf\n", volume);
   
   return 0;
}